Client/Application/QuickActions.swift

```diff
@@ -133,7 +133,10 @@ class QuickActions: NSObject {
         bvc.switchToTabForURLOrOpen(urlToOpen, isPrivileged: true)
     }
     
-    fileprivate func handleQRCode(withBrowserViewController bvc: BrowserViewController) {
-        bvc.openQRReader()
+    fileprivate func handleQRCode(with vc: QRCodeViewControllerDelegate & UIViewController) {
```

Comment:
I'm ambivalent about using algebraic types here. 

But standing ovation for using it to get rid of BVC.